reporting_guidelines.md

```diff
@@ -0,0 +1,51 @@
+# Reporting Guidelines
+
+If you believe someone is violating the [Code of Conduct](./code_of_conduct.md) we ask that you report it to the community admins by emailing osdghc@anitab.org.
+
+**All reports will be kept confidential.** In some cases we may determine that a public statement will need to be made. If that's the case, the identities of all victims and reporters will remain confidential unless those individuals instruct us otherwise.
+
+If you believe anyone is in physical danger, please notify appropriate emergency services first. If you are unsure what service or agency is appropriate to contact, include this in your report and we will attempt to notify them.
+
+In your report please include:
+
+* Your contact info for follow-up contact.
+* Names (legal, nicknames, or pseudonyms) of any individuals involved.
+* If there were other witnesses besides you, please try to include them as well.
+* When and where the incident occurred. Please be as specific as possible.
+* Your account of what occurred. 
+* If there is a publicly available record (e.g. a mailing list archive or a public IRC logger) please include a link.
+* Any extra context you believe existed for the incident.
+* If you believe this incident is ongoing.
+* Any other information you believe we should have.
+
+## What happens after you file a report?
+
+You will receive an email from the Anitab.org Open Source's Code of Conduct response team acknowledging receipt as soon as possible, but within 48 hours.
```

Comment:
Open Source Day's

Reply:
done + other places with `AnitaB.org's Open Source` to `AnitaB.org's Open Source Day`